pkg/stats/io_snapshots.go

```diff
@@ -64,35 +160,135 @@ func (h *snapshotHistory) SetOutput(id string, latestOutput json.Marshaler) {
 	h.latestOutput[id] = latestOutput
 }
 
-func (h *snapshotHistory) GetOutput() ([]byte, error) {
+func (h *snapshotHistory) GetOutputCopy() (map[string]interface{}, error) {
 	h.lock.RLock()
 	defer h.lock.RUnlock()
 	if h.latestOutput == nil {
-		return []byte{}, nil
+		return map[string]interface{}{}, nil
 	}
-	result := map[string]interface{}{}
-	for id, obj := range h.latestOutput {
-		jsn, err := obj.MarshalJSON()
-		if err != nil {
-			return nil, err
+	genericMaps, err := getGenericMaps(h.latestOutput)
+	if err != nil {
+		return nil, err
+	}
+	return genericMaps, nil
+}
+
+// DEPRECIATED - Use GetFilteredOutput or GetOutputCopy instead
+func (h *snapshotHistory) GetOutput() ([]byte, error) {
+	output, err := h.GetOutputCopy()
```

Comment:
Same Q as above, could we just wrap GetFilteredOutput with no filters and json format?